src/containers/Admin/EditTab/ScooterPanel/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback } from 'react'
+import { Checkbox, Fieldset } from '@entur/form'
+import { ScooterOperator, Scooter } from '@entur/sdk'
+
+import { toggleValueInList } from '../../../../utils'
+import { useSettingsContext } from '../../../../settings'
+
+import './styles.scss'
+import { Paragraph } from '@entur/typography'
+
+function ScooterPanel(props: Props): JSX.Element {
+    const [settings, { setHiddenOperators }] = useSettingsContext()
+    const { hiddenOperators = [] } = settings || {}
+
+    const { scooters } = props
+    const allOperators = ['voi', 'tier', 'lime', 'zvipp']
```

Comment:
Dette er ein konstant, så den kan ut av komponenten. Me pleier å bruke store bokstavar for konstantar. Og bruk helst ScooterOperator-enumen:

```
const ALL_OPERATORS = [ScooterOperator.VOI, ScooterOperator.TIER, ScooterOperator.LIME, ScooterOperator.ZVIPP]

function ScooterPanel(props: Props): JSX.Element { 
    ...
```

Reply:
Fixed